src/view/components/cpx/CpxImage.tsx

```diff
@@ -380,7 +380,7 @@ export const updateSwitch = (switchState: boolean): void => {
     if (switchElement && switchInner) {
         svg.addClass(switchInner, "sim-slide-switch-inner");
 
-        if (switchState) {
+        if (!switchState) {
             svg.addClass(switchInner, "on");
             switchInner.setAttribute("transform", "translate(-5,0)");
         } else {
```

Comment:
`
if (condition) {
doIfTrue();
} else {
doIfFalse();
}
` is easier to understand than `if (!condition) { doIfFalse(); } else { doIfTrue(); }`
I would swap the blocks.